hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -109,12 +111,29 @@ def create(project_root: Path,
         else:
             logging.info("Running inside AzureML.")
             logging.info("All results will be written to a subfolder of the project root folder.")
-            run_folder = project_root
-            outputs_folder = project_root / DEFAULT_AML_UPLOAD_DIR
-            logs_folder = project_root / DEFAULT_LOGS_DIR_NAME
+            amulet_keys_not_set = get_amulet_keys_not_set()
+            if len(amulet_keys_not_set) > 0:
+                logging.debug("Assumed not to be an Amulet job since the following environment variables are not set:"
+                              f" {amulet_keys_not_set}")
+                run_folder = project_root
+                outputs_folder = project_root / DEFAULT_AML_UPLOAD_DIR
+                logs_folder = project_root / DEFAULT_LOGS_DIR_NAME
+            else:
+                # Job submitted via Amulet
+                amlt_root_folder = Path(os.environ["AZURE_ML_INPUT_OUTPUT"])
+                project_name = os.environ["AZUREML_ARM_PROJECT_NAME"]
+                snapshot_dir = Path(os.environ["SNAPSHOT_DIR"])
```

Comment:
can we import the variable names from hi-ml-azure/src/health_azure/utils.py?

Reply:
Good point